qa/tasks/scripts/ganesha_smoke_test.sh

```diff
@@ -4,66 +4,233 @@
 #
 # runs on first client node
 #
-# args: --rgw, --mds, --ganesha-host=$IP
+# args: --rgw, --mds
 
 set -e
 
 NFS_MOUNTPOINT=/root/mnt
 
-GANESHAHOST=""
 MDS=""
 RGW=""
 while true ; do
     case "$1" in
-        --ganesha-host) GANESHAHOST=$(echo $GANESHAHOST | sed ...grab just the host part...) ; shift ;;
         --mds) MDS="$1" ; shift ;;
         --rgw) RGW="$1" ; shift ;;
         --) shift ; break ;;
         *) echo "Internal error" ; false ;;
     esac
 done
 
-echo "Running ganesha_smoke_test.sh with option(s) --ganesha-host=$GANESHAHOST $MDS $RGW"
+echo "Running ganesha_smoke_test.sh with option(s) [--mds] --[rgw]"
```

Comment:
The idea behind this line was to emit a message to the teuthology log saying how the script was run, so maybe this should be:

```
echo "Running ganesha_smoke_test.sh with option(s) $MDS $RGW" >/dev/null
```

Because `$MDS` will expand to "--mds" if that option was passed, and same with `$RGW`.

Alternatively, just drop this line altogether?

Reply:
ah, you're right. I misunderstood the purpose of that line.